spec/rails_helper.rb

```diff
@@ -48,6 +48,13 @@ def visit_sandbox
   visit page_path(id: 'sandbox')
 end
 
+def click_download_panel
+  expect(page).to have_css('.sul-embed-download-panel', visible: false)
+  page.find('button[data-sul-embed-toggle="sul-embed-download-panel"]', visible: true)
+  page.find('[data-sul-embed-toggle="sul-embed-download-panel"]', match: :first).click
+  expect(page).to have_css('.sul-embed-download-panel', visible: true)
+end
```

Comment:
I'm not so fond of embedding the expectations in this helper method;  I would be more in favor of having a test in a spec file with the two expect statements bracketing a call to a 2-line click-download-panel.   It would surface the test as a test, and it would avoid running the test for every click.

If this is something that the calling class needs to test ... then rename to click_toggable_download_panel   

Not a huge thing I feel strongly about ... just a thought.